package.json

```diff
@@ -9,7 +9,7 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "next": "14.2.4",
+    "next": "^14.2.5",
```

Comment:
Do not upgrade to the next version at this time to maintain consistency across multiple developers.